### ISSUE TITLE
Compare with other approaches

### ISSUE THREAD

**Issue by creatorrr** [huggingface/setfit#227]
Dumb question:

How does setfit compare with other approaches for sentence classification in low data settings? Two that may be worth comparing to:

- Various techniques for [augmented SBERT](https://www.sbert.netexamples/training/data_augmentation/README.html)
- Simple Contrastive Learning [SimCSE](https://github.com/princeton-nlp/SimCSE)

Pros and cons of these approaches? Thoughts?

**Comment by tomaarsen** [comment]
The augmented SBERT link is faulty: there's a `/` missing. The [augmented SBERT](https://www.sbert.net/examples/training/data_augmentation/README.html) link does work.

Those would be very interesting to compare against. I'm not sure if anyone has tried to do so yet.

**Comment by creatorrr** [comment]
Thanks for the correction. :)

**Comment by creatorrr** [comment]
Yes I agree. Is there some dataset that we could use as a good benchmark?
